src/main/groovy/pl/droidsonroids/gradle/pitest/PitestPlugin.groovy

```diff
@@ -213,16 +217,20 @@ class PitestPlugin implements Plugin<Project> {
             coverageThreshold.set(extension.coverageThreshold)
             mutationEngine.set(extension.mutationEngine)
             exportLineCoverage.set(extension.exportLineCoverage)
-//        jvmPath.set(extension.jvmPath)
+            jvmPath.set(extension.jvmPath)
```

Comment:
Hmm, it seems that in line 263 you still have:
```
            jvmPath = { extension.jvmPath }
```
It should be not needed. I believe I removed it in changes.

Reply:
See https://github.com/szpak/gradle-pitest-plugin/blob/aea257342e30c25d9814508172c4df527837a6bf/src/main/groovy/info/solidsoft/gradle/pitest/PitestPlugin.groovy#L185-L186